src/audio/copier/copier.c

```diff
@@ -1485,10 +1504,11 @@ static int copier_params(struct comp_dev *dev, struct sof_ipc_stream_params *par
 							 dev->pipeline,
 							 child_dev->frames,
 							 is_scheduling_source);
+
```

Comment:
it is common in SOF to place code like
```
ret = fn();
if (ret < 0)
	handle_error();
```
without empty lines between the first two lines

Reply:
ok, empty line removed.